src/pages/[project]/study-design/index.js

```diff
@@ -3,23 +3,61 @@ import { Layout } from "@/layouts/Layout";
 import {
   attributes,
   react as StudyContent,
-} from "@/content/pages/study-design.md";
+} from "@/content/pages/mmp-study-design.md";
+import {
+  attributes as lampAttributes,
+  react as StudyContentLamp,
+} from "@/content/pages/lamp-study-design.md";
 import Image from "next/image";
+import { withRouter } from "next/router";
 
-export default class StudyDesignPage extends Component {
+class StudyDesignPage extends Component {
   render() {
```

Comment:
Quick question, I noticed that some pages (like this one) use render() method while others use a function component with a direct return(). Is there a specific reason behind choosing one approach over the other? Just out of curiosity!

Reply:
Predates me, but I think it might have to do with the fact that this was migrated from another framework. @hetd54 any background to that?